features/step_definitions/admin_can_crud_article.rb

```diff
@@ -0,0 +1,39 @@
+Given("The following Admin exists") do |table|
+  table.hashes.each do |admin_user|
+    FactoryBot.create(:admin_user, admin_user)
+  end
+end
+
+Given("The following Article exists") do |table|
+  table.hashes.each do |article|
+    FactoryBot.create(:article)
+  end
+end
+
+When("I click dashboard link Articles") do
+  click_link('Articles')
+end
+
+When("I click admin article page link {string}") do |string|
+  click_link(string)
+end
+
+Then("I should be on {string} dashboard page") do |string|
+  visit new_admin_article_path
+end
+
+Then("I fill in field {string} with {string}") do |field, value|
+  fill_in(field, with: value)
+end
+
+Then("I click article link {string}") do |string|
+  click_link_or_button string
+end
+
+Then("I click article update button {string}") do |string|
+  click_link_or_button string
+end
+
+Then("I should be redirected to article edit page") do
+  visit edit_admin_article_path
+end
```

Comment:
I have a feeling that we already have a lot of these steps somewhere else in the code? 
Might wanna reuse the ones we already have! :)

Reply:
Good point. At this stage I would try to get the paths from a method using  a conditional (`case` or `if`)